test/e2e/run_networking_test.go

```diff
@@ -639,6 +639,22 @@ var _ = Describe("Podman run networking", func() {
 		Expect(create.ExitCode()).To(BeZero())
 	})
 
+	It("podman run in custom CNI network with --uidmap", func() {
+		netName := "podmantestnetwork2"
```

Comment:
Please use `stringid.GenerateNonCryptoID()` for the network name. The static network names are causing a lot of flakes.